backend/src/controllers/reviewController.ts

```diff
@@ -23,6 +25,37 @@ export const addReview = async (req: Request, res: Response) => {
   }
 }
 
+export const deleteReview = async (req: Request, res: Response) => {
+  const jwt = req.cookies.authToken;
+  if (!jwt) {
+    res.status(400).json({ message: 'Missing JWT cookie' });
+  }
+  let id: number;
+  try {
+    const payload = auth.verifyAndReadJWT(jwt);
+    id = payload.id;
+
+    const review = req.body;
+    if (!review || !review.id || !review.account_id) {
```

Comment:
all you need to delete is the reviewID. i'd just do the body here as expecting an ID in data. you'll get the account ID directly from the jwt